x/incentive/keeper/keeper_fees.go

```diff
@@ -0,0 +1,31 @@
+package keeper
+
+import (
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/cosmos/cosmos-sdk/x/distribution/types"
+)
+
+// Move gas fees collected to incentive module
+func (k Keeper) CollectGasFeesToIncentiveModule(ctx sdk.Context) sdk.Coins {
+	// fetch and clear the collected fees for distribution, since this is
+	// called in BeginBlock, collected fees will be from the previous block
+	// (and distributed to the previous proposer)
+	feeCollector := k.authKeeper.GetModuleAccount(ctx, k.feeCollectorName)
+	feesCollectedInt := k.bankKeeper.GetAllBalances(ctx, feeCollector.GetAddress())
+	// feesCollected := sdk.NewDecCoinsFromCoins(feesCollectedInt...)
+
+	// transfer collected fees to the distribution module account
+	err := k.bankKeeper.SendCoinsFromModuleToModule(ctx, k.feeCollectorName, types.ModuleName, feesCollectedInt)
+	if err != nil {
+		panic(err)
+	}
+
+	return feesCollectedInt
+}
+
+// Pull DEX revenus collected to incentive module
+func (k Keeper) CollectDEXRevenusToIncentiveModule(ctx sdk.Context) sdk.Coins {
+	// transfer collected fees to the distribution module account
```

Comment:
Can you add TODO word here so that next time when we check what's missing, it can be easily detected.

Reply:
Okay got it. Will do this in the next PR.